extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrPhotosImporter.java

```diff
@@ -172,14 +176,20 @@ private void importSinglePhoto(UUID id, PhotoModel photo) throws FlickrException
 
   private String uploadPhoto(PhotoModel photo, UUID jobId) throws IOException, FlickrException {
     BufferedInputStream inStream = imageStreamProvider.get(photo.getFetchableUrl());
+
+    // TODO: do we want to keep COPY_PREFIX?  I think not
+    String photoTitle =
+        Strings.isNullOrEmpty(photo.getTitle()) ? "" : COPY_PREFIX + photo.getTitle();
```

Comment:
Same question regarding importing empty titles or if we should add untitled.

Reply:
Flickr does support empty titles.